src/utils/editor/index.js

```diff
@@ -180,3 +180,16 @@ export function getTextForLine(codeMirror, line) {
 export function getCursorLine(codeMirror) {
   return codeMirror.getCursor().line;
 }
+
+export function isEditorVisible(mutationsList) {
+  for (const mutation of mutationsList) {
```

Comment:
how do we know this is the editor that is no longer collapsed?

Reply:
Good call, I just checked, and this fix doesn't work for all the tabs, that collapse is triggered for the console which has its own `<box ...>` element. The other tabs seem to be in `<vbox ...>`'s in the same box as the debugger. It think it would be better to observe if the debugger tab is selected or not. Which is set in the debugger's `<vbox ...>`.

![image](https://user-images.githubusercontent.com/14250545/37222755-7b3dc548-238b-11e8-9e65-eadcf229ed7c.png)